app/services/hyrax/workflow/notification_service.rb

```diff
@@ -51,10 +51,10 @@ def recipients(notification)
         case entity.workflow_state.name
         when 'pending_review'
           # notify the managers to review the deposit (CC depositors)
-          {
-            to: entity.workflow.permission_template.agent_ids_for(access: 'manage',  agent_type: 'user'),
-            cc: entity.workflow.permission_template.agent_ids_for(access: 'deposit', agent_type: 'user')
-          }
+          permissions = entity.workflow.permission_template
+          managers = Array.wrap(permissions.agent_ids_for(access: 'manage', agent_type: 'user'))
+          depositors = Array.wrap(permissions.agent_ids_for(access: 'deposit', agent_type: 'user'))
+          { 'to' => managers, 'cc' => depositors }
```

Comment:
This change may be the more pertinent one. It could be the underlying code cares about what form its keys are in.